TOMP-API.yaml

```diff
@@ -1210,7 +1204,7 @@ components:
           type: array
           items:
             type: string
-            enum: [FROM-ADDRESS, TO-ADDRESS, BIRTHDATE, EMAIL, PERSONAL-ADDRESS]
+            enum: [FROM-ADDRESS, TO-ADDRESS, BIRTHDATE, EMAIL, PERSONAL-ADDRESS, GENDER, PHONE-NUMBERS, LICENSES, BANK-CARDS, DISCOUNT-CARDS, TRAVEL-CARDS, ID-CARDS, CREDIT-CARDS]
```

Comment:
I'm not exactly sure what this object is for, but I see a lot of personal data being transferred between parties here. Isn't the idea of TOMP to NOT do that?

Reply:
Agree, seems to me that some properties could be removed or at least made optional, for example: gender,  bank/credit cards

Reply:
he guys, it facilitates the scenario that a TO can request a some personal data. This can be done by the the requestBookingDataCondition, where some of these enums can be used. It's just as requested in the issue by Jef and Ross. They are all optional, you don't have to use the requestBookingDataCondition. It should only be used when the TO demands it.